src/main/kotlin/pet/project/app/controller/BookController.kt

```diff
@@ -42,12 +40,16 @@ class BookController(private val bookService: BookService) {
         return updatedBook.toDto()
     }
 
+    data class AmountRequest(
```

Comment:
```suggestion
    data class UpdateAmountRequest(
```
move the class to separate file, please